gradle.properties

```diff
@@ -4,8 +4,12 @@ org.gradle.jvmargs=-Xmx3G
 org.gradle.daemon=true
 org.gradle.parallel=true
 org.gradle.caching=true
-org.gradle.configuration-cache=false
+org.gradle.configuration-cache=true
 org.gradle.debug=false
+#org.gradle.warning.mode=fail
+
+moddevgradle_plugin_version=2.0.43-beta
```

Comment:
Please add a comment with the format `# lib: <artifact ID>` above each version that represents a tool that's not directly referenced in a configuration in `build.gradle`. That will allow Renovate to catch them once I set up a custom manager.